astroquery/sha/core.py

```diff
@@ -144,8 +145,13 @@ def parse_line(line, cs=cs):
     dtypes = _map_dtypes(type_names, field_widths)
     # To table
     # transpose data for appropriate table instance handling
-    t = Table(list(zip(*data)), names=col_names, dtype=dtypes)
-    return t
+    if len(data) > 0:
+        t = Table(list(zip(*data)), names=col_names, dtype=dtypes)
+        return t
+    else:
+        warnings.warn(NoResultsWarning("No matching rows were found in the query."))
+        t = Table()
+        return t
```

Comment:
Suggest moving the "return t" outside of the if/else.

Reply:
Sorry. This was indent issue!